src/reports/components/report-sections/link-script-provider.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export const addEventListenerForLink = (doc: Document) => {
+    const targetPageLink = doc.getElementById('target-page-link');
+
+    targetPageLink.addEventListener('click', event => {
+        const result = confirm(
```

Comment:
Here, `confirm` is being use as a global dependency which is not ideal. One way to solve this is to do the same you are doing with the `doc: Document` param. That way, you can mock the function on the unit tests.